promise-types/symlinks/symlinks.py

```diff
@@ -0,0 +1,99 @@
+import os
+from cfengine import PromiseModule, ValidationError, Result
+
+
+class SymlinksPromiseTypeModule(PromiseModule):
+
+    def __init__(self, **kwargs):
+        super(SymlinksPromiseTypeModule, self).__init__(
+            name="symlinks_promise_module",
+            version="0.0.1",
+            **kwargs,
+        )
+
+        def is_absolute_dir(v):
+            if not os.path.isabs(v):
+                raise ValidationError("must be an absolute path, not '{v}'".format(v=v))
+            if not os.path.exists(v):
+                raise ValidationError("dir must exists")
```

Comment:
```suggestion
                raise ValidationError("directory must exists")
```
Maybe you can spell it out, if it's printed to the user